lockpick-libzfs-abi.sh

```diff
@@ -0,0 +1,210 @@
+#! /usr/bin/env bash
+
+# Try to "lockpick" the settings relevant for libzfs.jar on this host OS
+# Note that the "correct answer" may change across OS updates... as well
+# as libzfs.jar evolution. Requires sources, "mvn" and JDK at this time,
+# and you may have to allow tests in pom.xml (set "<skip>" to "false").
+# Note it is likely to leave around Java coredump files, or at least logs
+# of those with stack traces.
+#
+# TODO: Add an option to libzfstest to run just a certain native routine.
+#
+# Copyright (C) 2017 by Jim Klimov
+#
+# Recommended usage to track down issues:
+#  VERBOSITY=high ./lockpick-libzfs-abi.sh | tee "lock.`date +%s`.log"
+#
+# See also some docs:
+#  http://maven.apache.org/surefire/maven-surefire-plugin/examples/single-test.html
+#
+
+# Bashism to allow pipes to fail not only due to last called program
+# Also, below, bash associative arrays are used
+set -o pipefail
+
```

Comment:
`set -e`

Just to make sure the script fails on unhandled error (e.g. no permissions for ulimit)